README.md

```diff
@@ -1,82 +1,41 @@
-Takeaway Challenge
-==================
-```
-                            _________
-              r==           |       |
-           _  //            |  M.A. |   ))))
-          |_)//(''''':      |       |
-            //  \_____:_____.-------D     )))))
-           //   | ===  |   /        \
-       .:'//.   \ \=|   \ /  .:'':./    )))))
-      :' // ':   \ \ ''..'--:'-.. ':
-      '. '' .'    \:.....:--'.-'' .'
-       ':..:'                ':..:'
-
- ```
+# Takeaway Challenge #
 
-Instructions
--------
+  This program replicates a takeaway delivery service (specifically the online aspect of ordering food from a menu).
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+  I approached this challenge by using TDD and debugging with Rspec tests to reach my final version. I wrote out a Rspec test that described my aim and then followed the error messages to create source-code that corresponded to those errors.
 
-Task
------
+  It acts as practice for my Ruby and Rspec skills, as well as introducing new skills (such as using Twilio to send texts using my code).
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+## What criteria should this program satisfy?
```

Comment:
Clear and defined points on readme > it shows you have really thought about what this challenge offers for your learning path. Will try and do this in mine.